jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java

```diff
@@ -23,117 +23,150 @@ protected T table() {
         element.refresh();
         return element;
     }
+    
     /**
      * Check that the table is empty
      */
     @JDIAction("Assert is '{name}' empty")
-    public A empty() {
-        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is empty"));
+    public A empty(String... messages) {
+        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is empty"), messages);
         return (A) this;
     }
-
+    
     /**
      * Check that the table isn't empty
      */
     @JDIAction("Assert is '{name}' not empty")
-    public A notEmpty() {
-        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is not empty"));
+    public A notEmpty(String... messages) {
+        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is not empty"), messages);
         return (A) this;
     }
-
+    
     /**
      * Match passed value with table size
+     *
      * @param condition to compare
      */
     @JDIAction("Assert that '{name}' size {0}")
-    public A size(Matcher<Integer> condition) {
-        jdiAssert(table().count(), condition);
+    public A size(Matcher<Integer> condition, String... messages) {
+        jdiAssert(table().count(), condition, messages);
         return (A) this;
     }
-
+    
     /**
      * Match passed value with table size
+     *
      * @param size to compare
      */
     @JDIAction("Assert that '{name}' size {0}")
-    public A size(int size) {
-        return size(Matchers.is(size));
+    public A size(int size, String... messages) {
+        return size(Matchers.is(size), messages);
+    }
+    
+    /**
+     * Match passed value with table rows count
+     * @param condition to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(Matcher<Integer> condition) {
+        jdiAssert(table().count(), condition);
+        return (A) this;
     }
 
+    /**
+     * Match passed value with table rows count
+     * @param count to compare
+     */
+    @JDIAction("Assert that '{name}' count {0}")
+    public A count(int count) {
+        return count(Matchers.is(count));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)